src/integrations/const.ts

```diff
@@ -0,0 +1,4 @@
+const ProductionUrl = "https://developer-platform-api.crypto.com";
+const LocalUrl = "http://localhost:8091";
+const BaseUrl = process.env.NODE_ENV === 'development' ? LocalUrl : ProductionUrl;
+export { BaseUrl };
```

Comment:
Because this will be a published npm package so it might not be a good idea to rely on `NODE_ENV` as ideally we want users to be able to use this package out of the box without any configuration, maybe we keep these configuration only for local testing and not commit it to main?

Reply:
that is correct